phpstan.neon

```diff
@@ -18,6 +18,7 @@ parameters:
 		- tests/*/data-attributes/*
 		- tests/*/data-php-*/*
 		- tests/Rules/Doctrine/ORM/entity-manager.php
+		- tests/Infection/
```

Comment:
Why exclude it?

Reply:
Yeah I can imagine a repository similar to https://github.com/phpstan/build-cs once we need this in multiple repositories 😊